c/frame/replace.cc

```diff
@@ -252,7 +255,7 @@ void ReplaceAgent::parse_x_y(const Arg& x, const Arg& y) {
 //
 // Here we analyze the input lists `vx`, `vy` and split them into 4 sublists
 // according to their elements types. We also do further verification that the
-// types of elements in vectors `vx`, `vy` match, and that there no duplicates.
+// types of elements in vectors `vx`, `vy` match, and that there is no duplicates.
```

Comment:
*are

Reply:
I guess both forms can be used:
"[there is no duplicates](https://www.google.com/search?q=%22there+is+no+duplicates%22)" About 146,000 results 
"[there are no duplicates](https://www.google.com/search?q=%22there+are+no+duplicates%22)" About 155,000 results

Reply:
After comparing the sources reliability, changed this to "are".